dash_bio/component_factory/_clustergram.py

```diff
@@ -203,75 +199,72 @@ def Clustergram(
-    scipy.cluster.hierarchy.linkage). (Default: sch.linkage)
-- color_threshold (dict; optional): Maximum linkage value for which
-    unique colors are assigned to clusters; 'row' for rows, and 'col'
-    for columns. (Default: {'row': 0, 'col': 0})
-- optimal_leaf_order (bool; optional): Enabling/disabling of the
+    row-clustered data.
+- row_dist (string; default 'euclidean'): String specifying the
+    distance metric for rows. It will be passed as the argument
+    'metric' into the function specified in dist_fun (see
+    scipy.spatial.distance.pdist).
+- col_dist (string; default 'euclidean'): String specifying the
+    distance metric for columns. It will be passed as the argument
+    'metric' into the function specified in dist_fun (see
+    scipy.spatial.distance.pdist).
+- dist_fun (function; default scipy.spatial.distance.pdist): Function
+    to compute the pairwise distance from the observations (see
+    scipy.spatial.distance.pdist).
+- link_fun (function; default scipy.cluster.hierarchy.linkage): Function to
+    compute the linkage matrix from the pairwise distances (see
+    scipy.cluster.hierarchy.linkage).
+- color_threshold (dict; default {'row': 0, 'col': 0}): Maximum
+    linkage value for which unique colors are assigned to clusters;
+    'row' for rows, and 'col' for columns.
+- optimal_leaf_order (bool; default false): Enabling/disabling of the
     option to determine leaf order that maximizes similarity between
-    neighboring leaves. (Default: False)
-- color_map (list; optional): The colorscale for the heatmap. Each
-    list element contains two elements; the first element refers to
-    the portion of the maximum data point under which a cell will be
-    colored, and the second element refers to the color. e.g., a
-    colorscale [[0.0, 'white'], [0.5, 'gray'], [1.0, 'black']] mean
-    that for all cells with a value in the 50th or lower percentile of
-    the dataset, the color on the heatmap would be white; all cells
-    with a value in the 50th or higher percentile, excluding the 100th
-    percentile, would be gray; and the cell(s) in the 100th percentile
-    would be colored black. (Default: [[0.0, 'rgb(255,0,0)'], [0.5,
-    'rgb(0,0,0)'], [1.0, 'rgb(0,255,0)']])
+    neighboring leaves.
+- color_map (list; default [[0.0, 'rgb(255,0,0)'], [0.5,
+    'rgb(0,0,0)'], [1.0, 'rgb(0,255,0)']]): The colorscale for the
+    heatmap. Each list element contains two elements; the first
+    element refers to the portion of the maximum data point under
+    which a cell will be colored, and the second element refers to the
+    color, e.g., a colorscale [[0.0, 'white'], [0.5, 'gray'], [1.0,
+    'black']] means that for all cells with a value in the 49th
+    percentile, the color on the heatmap would be
+    white; all cells at the 50th or higher percentile,
+    excluding the 100th percentile, would be gray; and the cell(s) at
+    the 100th percentile would be colored black.
```

Comment:
Good, we got the percentiles straight in the paragraph above. Now, it would be helpful to keep only one phrasing for all three cases (colours), where this phrasing may 'shrink' as we move forward, but not change completely, e.g.:

* "for values in the 49th percentile, cells would be colored white on the heatmap; for values at the 50th or higher percentile, excluding the 100th percentile, cells would be colored gray; and for values at the 100th percentile, black."
* "for cells with a value in the 49th percentile, the color on the heatmap would be white; for cells with a value at the 50th or higher percentile, excluding the 100th percentile, the color would be gray; and for cells with a value at the 100th percentile, black."

Not going from "cells" to "values," from "color would be" to "would be colored" helps with cognitive load.